src/screen/dashboard.rs

```diff
@@ -1126,12 +1095,13 @@ impl Dashboard {
                         &self.side_menu,
                         config,
                         theme,
-                        main_window,
+                        window != self.main_window(),
                     )
                 })
                 .spacing(4)
                 .on_click(pane::Message::PaneClicked),
-            )
+                pane::Message::MaintainFocus,
+            ))
```

Comment:
We already have `on_click(pane::Message::PaneClicked)`, why do we need this other click / event? Just use `pane::Message::PaneClicked` to refocus the text input